client/src/common/components/Activities/components/Activity.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Fragment, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import { dateFromString } from 'common/utils/helpers';
+import Avatar from 'common/components/Avatar';
+
+class Activity extends PureComponent {
+  renderCohortInfo = () => {
+    const {
+      activity: { cohort }
+    } = this.props;
+
+    if (cohort) {
+      const { cohortId, cohortName } = cohort;
+      return (
+        <Fragment>
+          {' in '}
+          <Link className="activity__link" to={`/cohort/${cohortId}`}>
+            {cohortName}
+          </Link>
+          {' cohort'}
+        </Fragment>
+      );
+    }
+  };
+
+  renderListInfo = () => {
+    const {
+      activity: { list }
+    } = this.props;
+
+    if (list) {
+      const { listId, listName } = list;
+      return (
+        <Fragment>
+          <Link className="activity__link" to={`/sack/${listId}`}>
+            {listName}
+          </Link>
+        </Fragment>
+      );
+    }
+  };
+
+  render() {
+    const {
+      activity: {
+        actor: { actorAvatarUrl, actorName },
```

Comment:
why actor?

Reply:
I found this naming convention in one solution regarding activities in bank application. An actor is a user who performed an action. Of course, it could be a simple name: user. I used "actor" to distinguish it from the second user we can have in activity data: eg. a user who was edited. If you think it's strange and unclear I can't change it

Reply:
why not just 
`performer` ? 
`performerAvatrUrl `,  `performerName`